modules/browser/BrowserCapabilities.js

```diff
@@ -102,6 +103,7 @@ export default class BrowserCapabilities extends BrowserDetection {
      */
     isSupported() {
         return (this.isChromiumBased() && this._getChromiumBasedVersion() >= MIN_REQUIRED_CHROME_VERSION)
+            || (this.isSafari() && this._getSafariVersion() >= MIN_REQUIRED_SAFARI_VERSION)
```

Comment:
Watch out, `isWebKitBased` will return true.

Reply:
i tested in safari 15 and it didnt..

Reply:
updated